### PR TITLE
Philips LTC005

### PR DESCRIPTION
**Pull request by yhur**
I noticed this Philips Hue is not recognized by the Zigbee2MQTT, so did some comparison, came up with this modification on the philips.js. Please take a look and merge it if there is no problem.

[Philips Hue Within Ceiling Lamp](https://www.philips-hue.com/ko-kr/p/hue-white-ambiance-within-ceiling-light/4505548C5)
Screen Shot 2021-09-20 at 11 12 17 PM

### CODE REVIEW

File: devices/philips.js [hunk 1]
```diff
@@ -1040,6 +1040,14 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp(),
         ota: ota.zigbeeOTA,
     },
+    {                                           
+        zigbeeModel: ['LTC005'],                      
+        model: '3216131P6',                                                         
+        vendor: 'Philips',
+        description: 'Hue ambiance ceiling',                
+        meta: {turnsOffAtBrightness1: true},                    
+        extend: hueExtend.light_onoff_brightness_colortemp(),
```
Comment (Koenkk) [inline devices/philips.js:1049]: Please add the color temperature range as described on https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html#31-retrieving-color-temperature-range-only-required-for-lights-which-support-color-temperature

Comment (yhur) [reply devices/philips.js]: Hi, @Koenkk,
Thank you for the review and the link to the information. I just updated and pushed to my repo after querying through the mqtt.

Would you take a look and let me know if you want me to reopen this or close this one and open a new PR?

File: devices/philips.js [hunk 2]
```diff
@@ -1065,7 +1073,6 @@ module.exports = [
         description: 'Hue white ambiance Aurelle square panel light',
         meta: {turnsOffAtBrightness1: true},
         extend: hueExtend.light_onoff_brightness_colortemp(),
-        ota: ota.zigbeeOTA,
```
Comment (Koenkk) [inline devices/philips.js:1068]: Why was this removed?

File: devices/philips.js [hunk 3]
```diff
@@ -1040,6 +1040,14 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp(),
         ota: ota.zigbeeOTA,
     },
+    {                                           
+        zigbeeModel: ['LTC005'],                      
+        model: '3216131P6',                                                         
+        vendor: 'Philips',
+        description: 'Hue ambiance ceiling',                
+        meta: {turnsOffAtBrightness1: true},                    
+        extend: hueExtend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 454]}),
```
Comment (Koenkk) [inline devices/philips.js:1049]: Please also add `ota: ota.zigbeeOTA,` here

File: devices/philips.js (hunk 2 above)
Comment (yhur) [reply devices/philips.js:1068]: The reason I removed is as follows.
1. To code, I talked with Sygnify(Philips Hue) but couldn’t get the proper information from them.
2. And they said this model, ie LTC005 is withdrawn from the market so no more development.
3. So I played with the definitions for other Synify models to come up with working set for LTC005
4. I selected 3216131P6 definition as the base and modified since the icon is similar to LTC005 
5. Since I couldn’t get no development/patch related information from Sygnify, I removed the OTA to prevent the wrong update.

File: devices/philips.js (hunk 3 above)
Comment (Koenkk) [reply devices/philips.js:1049]: Makes sense, thanks!
